_overviews/compiler-options/index.md

```diff
@@ -44,34 +40,41 @@ This page comes to the rescue for the people to find&hellip;
 ```bash
 scalac [ <options> ] <source files>
 ```
+Boolean flags are specified in the usual way:
+
+`scalac -Werror -Xlint Hello.scala`
+
+Options that require arguments use "colon" syntax:
+
+`scalac -Vprint:parser,typer`
 
-E.g. `scalac -encoding utf8 -Xfatal-warnings Hello.scala`
+Options that take just a single argument accept traditional syntax:
 
-Default paths can be listed by running a command line tool:
+`scalac -d /tmp`
+
+Default paths can be listed by running a tool in the distribution:
 ```
 scala scala.tools.util.PathResolver [ <options> ]
 ```
 
-
-
 ### Use compiler options with sbt
 
-
+Here is a typical configuration of the `scalacOptions` setting in `sbt`:
 
 ```scala
 scalacOptions ++= Seq(
-  "-encoding", "utf8", // Option and arguments on same line
-  "-Xfatal-warnings",  // New lines for each options
-  "-deprecation",
-  "-unchecked",
+  "-encoding", "utf8",          // if an option takes an arg, supply it on the same line
+  "-feature",                   // put the next option on a new line
```

Comment:
The newline is just for readability and not illegal? Perhaps change the comment then to:
`// for better readability, put the next option on a new line`

Reply:
it improves writability, so I said something like that. Autocorrect says "irritability", which is closer to the truth.